package.json

```diff
@@ -60,15 +61,22 @@
   "devDependencies": {
     "babel-cli": "^6.6.5",
     "babel-eslint": "^5.0.4",
+    "babel-jest": "^16.0.0",
     "babel-plugin-rewire": "1.0.0-beta-2",
     "deep-equal": "^1.0.1",
     "eslint": "^1.10.3",
     "eslint-config-airbnb": "^3.1.0",
     "eslint-plugin-react": "^3.14.0",
     "expect.js": "^0.3.1",
     "foreman": "^1.4.1",
+    "jest": "^16.0.2",
     "mocha": "^2.3.3",
```

Comment:
oh, weird, I thought `npm uninstall` would do that, guess I had to save or something